public/css/memo-admin.scss

```diff
@@ -254,52 +254,57 @@ $btn-link-height: 20px;
 
 // ------- MEMO MENU --------- //
 
-.select-semester {
-  width: 100%;
-  max-width: 370px;
-  margin-top: -30px;
-  margin-bottom: 20px;
-  & > .btn {
-    background-color: #f6f6f6;
-    border: 1px solid $light-grey;
-    display: flex;
-    justify-content: space-between;
-    padding: 0;
-    width: 100%;
-    & > span:first-child {
-      margin: 6px;
-      max-height: 28px;
-      overflow-y: hidden;
-      width: 100%;
-      text-align: left;
-    }
-    .caretholder {
-      background-color: $kth-blue;
-      border-bottom-right-radius: 0.25rem;
-      border-top-right-radius: 0.25rem;
-      border: 1px solid $kth-blue;
-      width: 42px;
-      height: 42px;
-      margin: -1px 0;
-      position: relative;
+// .select-semester {
```

Comment:
This commented-out code should be deleted to keep the file clean and dev-friendly.